db/migrate/20160414094300_change_capacity_to_hash_from_persistent_volume.rb

```diff
@@ -0,0 +1,56 @@
+class ChangeCapacityToHashFromPersistentVolume < ActiveRecord::Migration[5.0]
+  class ContainerVolume < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class PersistentVolume < ContainerVolume
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :capacity
+
+    IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+    def self.parse_iec_number(value)
```

Comment:
This part is a must, look at http://makandracards.com/makandra/15575-how-to-write-complex-migrations-in-rails "Embed models into your migrations" section.